rpm/python-cvmfsutils.spec

```diff
@@ -3,9 +3,9 @@
 
 Summary: Inspect CernVM-FS repositories
 Name: python-cvmfsutils
-Version: 0.5.0
+Version: %{version}
```

Comment:
I need there to be a spec file that does not require any macros passed in, in order to build this on the OpenSUSE Build System for the cvmfs-contrib repo.

I do like the fact that there's a single source for the version, but the only way I can see to do that is if you rename this file to python-cvmfsutils.spec.in and have the python-cvmfsutils.spec file generated with the version number appl,ied.  Perhaps it is not worth that, and we could just let that one little piece of information be duplicated.  There could be a ci check to make sure that they're identical so it's not accidentally forgotten.